test/e2e-go/features/incentives/mining_test.go

```diff
@@ -105,6 +127,7 @@ func TestBasicPayouts(t *testing.T) {
 		default:
 			a.Fail("bad proposer", "%v proposed", block.Proposer)
 		}
+		fixture.WaitForRoundWithTimeout(status.LastRound + 1)
```

Comment:
is this just a sanity check that the chain is still moving?

Reply:
It keeps the loop from running over and over on the same round by always waiting for the next one.